packages/core/src/scuttle.js

```diff
@@ -0,0 +1,181 @@
+/**
+ * @typedef {object} ScuttleOpts
+ * @property {boolean} enabled - Whether scuttling is enabled or not.
+ * @property {Array<string|RegExp>} exceptions - List of properties to exclude from scuttling.
+ * @property {string} scuttlerName - Name of the scuttler function to use.
+ * @property {(globalRef: object, scuttle: Function) => void} scuttlerFunc - Scuttler function to use.
+ */
+
+/**
+ * @typedef {object} GlobalRef
+ * @property {object} [globalThis] - Reference to the global object.
+ */
+
+const { Object, Array, Error, RegExp, Set, console, Proxy, Reflect } =
+  globalThis
+
+const {
+  assign,
+  getOwnPropertyNames,
+  getOwnPropertyDescriptor,
+  create,
+  defineProperty,
+} = Object
+
+const { isArray, from } = Array
+
+const { getPrototypeOf } = Reflect
+
+const { warn } = console
```

Comment:
I can't recall, but the `console` methods don't need the `console` context, do they?

Reply:
![Screenshot 2024-01-04 at 16 26 18](https://github.com/LavaMoat/LavaMoat/assets/13243797/8434c454-88d3-44a2-b8cf-935bad36d198)